test/evergreen.yml

```diff
@@ -1667,8 +1667,7 @@ buildvariants:
   run_on:
   - rhel80-test
   expansions:
-    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
-    posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-python --enable-zlib --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL CFLAGS="-fPIC -ggdb"
+    test_env_vars: LD_PRELOAD=/usr/local/lib/libeatmydata.so PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
```

Comment:
Can you please merge the existing value of `posix_configure_flags` to the new value of `test_env_vars`?

Reply:
No, we shouldn't do that. We have discussed this and we decided to keep them separate.

Reply:
But this change looks like if you have removed the `posix_configure_flags` all together?

Reply:
That means it will use the default value.